spec/function.dd

```diff
@@ -4230,3 +4248,7 @@ Macros:
         TITLE=Functions
         ASSIGNEXPRESSION=$(GLINK2 expression, AssignExpression)
         CHECK=&#10004;
+        REFP=$(LINK2 #ref-params, `ref`)
+        RETURN_REF=$(LINK2 #return-ref-parameters, `return ref`)
+        SCOPE=$(LINK2 #scope-parameters, `scope`)
+        RETURN_SCOPE=$(LINK2 #return-scope-parameters, `return scope`)
```

Comment:
```suggestion
        REFP=$(LINK2 #ref-params, $(D ref))
        RETURN_REF=$(LINK2 #return-ref-parameters, $(D return ref))
        SCOPE=$(LINK2 #scope-parameters, $(D scope))
        RETURN_SCOPE=$(LINK2 #return-scope-parameters, $(D return scope))
```

Reply:
The back tick does the same thing as $(D ).

Reply:
False.